### PR TITLE
Refactor function remove_range

### LINKED ISSUE #48: Refactor function remove_range
For P+ we need to refactor function in order to decrease CC.

### PR DESCRIPTION
**Pull request by capitanu**
Fixes #48

### CODE REVIEW

File: manual_evaluations/part2_coverage/refactored_functions/linkedlist/remove_range.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+def remove_range(head, start, end):
+
+    # pi = 0
+    # s = 0
+
+    branchCoverage = [False, False, False, False, False, False, False]    
+    
+    assert(start <= end)
+    assert(0 <= start)
+    assert(0 <= end)
```
Comment (ArnPellesGit) [inline manual_evaluations/part2_coverage/refactored_functions/linkedlist/remove_range.py:33]: What do these assertions do?

Comment (ArnPellesGit) [reply manual_evaluations/part2_coverage/refactored_functions/linkedlist/remove_range.py:33]: Are they basic input checks? Like to see that input's valid, i.e. end node is after start, start and end > 0?

**Review by ArnPellesGit — APPROVED** [review]
LGTM

File: manual_evaluations/part2_coverage/refactored_functions/linkedlist/remove_range.py (hunk 1 above)
Comment (ArnPellesGit) [reply manual_evaluations/part2_coverage/refactored_functions/linkedlist/remove_range.py:33]: And if that's the case, isn't assert() just a fancy "if" statement then? And if that is the case, then the CC should increase, no?

Comment (capitanu) [reply manual_evaluations/part2_coverage/refactored_functions/linkedlist/remove_range.py:33]: assert is in fact a fancy if statement, but that throws errors, so if PI is increased then S is increased as well and they sort of cancel out. One could just omit them imho.

Comment (ArnPellesGit) [reply manual_evaluations/part2_coverage/refactored_functions/linkedlist/remove_range.py:33]: I agree with this

**Pull request merged**